packages/big-design/src/components/Popover/Popover.tsx

```diff
@@ -115,7 +115,8 @@ const InternalPopover: React.FC<InternalPopoverProps> = ({
   });
 
   useEffect(() => {
-    const prevFocus = previousFocus.current as HTMLElement;
+    // @ts-expect-error current is HTMLElement
+    const prevFocus: HTMLElement = previousFocus.current;
 
     return () => {
       if (prevFocus && typeof prevFocus.focus === 'function') {
```

Comment:
We could check to see if `prevFocus` is an instance of `HTMLElement` instead 🤔

Reply:
this? 
```suggestion
     if (prevFocus && prevFocus instanceof HTMLElement && typeof prevFocus.focus === 'function') {
        prevFocus.focus();
      }
```